### PR TITLE
yoav - Discretize

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

**Review by YoniKiriaty — APPROVED** [review]

**Comment by ItayKauff** [comment]
code sucks
gg stands for get good

File: src/main/java/frc/robot/subsystems/swerve/Swerve.java [hunk 1]
```diff
@@ -350,7 +350,33 @@ private ChassisSpeeds powersToSpeeds(double xPower, double yPower, double thetaP
      */
     // todo - maybe move some of work to SwerveMath class
     private ChassisSpeeds discretize(ChassisSpeeds chassisSpeeds) {
-        return ChassisSpeeds.discretize(chassisSpeeds, RoborioUtils.getCurrentRoborioCycleTime());
```
Comment (ItayKauff) [inline src/main/java/frc/robot/subsystems/swerve/Swerve.java:353]: y delete
looks good

File: src/main/java/frc/robot/subsystems/swerve/Swerve.java [hunk 2]
```diff
@@ -350,7 +350,33 @@ private ChassisSpeeds powersToSpeeds(double xPower, double yPower, double thetaP
...
+        ChassisSpeeds current = getSelfRelativeVelocity();
+
+        double driveMagnitude = Math.sqrt(Math.pow(current.vxMetersPerSecond, 2) + Math.pow(current.vyMetersPerSecond, 2));
```
Comment (ItayKauff) [inline src/main/java/frc/robot/subsystems/swerve/Swerve.java:355]: stop speaking magic words magic man

File: src/main/java/frc/robot/subsystems/swerve/SwerveConstants.java [hunk 3]
```diff
@@ -28,12 +30,14 @@ public class SwerveConstants {
 
 
     public static final double DRIVE_NEUTRAL_DEADBAND = 0.2;
-    public static final double ROTATION_NEUTRAL_DEADBAND = 0.2;
-
+    public static final double ROTATION_NEUTRAL_DEADBAND = 0.2;//todo - rot2d
 
+    //todo - why affecting max rot
     public static final double MAX_SPEED_METERS_PER_SECOND = 5.5;
+    //todo - why when increasing max on run gets higher
```
[the remark this answers is not in the document]
Comment (ItayKauff) [reply src/main/java/frc/robot/subsystems/swerve/SwerveConstants.java:37]: please use fortnite terms

**Review by ItayKauff — CHANGES_REQUESTED** [review]

**Review by YoniKiriaty — DISMISSED** [review]

**Pull request closed without merging**